pdf/creator/table.go

```diff
@@ -606,8 +617,15 @@ func (table *Table) NewCell() *TableCell {
 	cell.borderColorRight = model.NewPdfColorDeviceRGB(col.ToRGB())
 	cell.borderColorTop = model.NewPdfColorDeviceRGB(col.ToRGB())
 
-	cell.rowspan = 1
-	cell.colspan = 1
+	// Set column span
+	if colspan <= 0 {
+		colspan = 1
+	}
+	if colspan > table.cols {
+		colspan = table.cols
```

Comment:
What if the cell is at the last column?  Would it no then go outside the table if colspan is table.cols ?
Seems like in that case it should be 1?  So should be if colspan > table.cols - cell.col ?

Reply:
Yes, I guess we could move the cell on the next row if it does not fit in the current one. However, it may produce some unexpected behavior. Another approach would be to limit the column to the colspan that is left on the current row (colspan = table.cols - cell.col).

Reply:
Yes.  Limiting to colspan = table.cols - cell.col sounds good and outputing a debug warning